### PR TITLE
task/WP-205: Add message for submitting >5 entities

### PR DESCRIPTION
**Pull request by edmondsgarrett**
## Overview
Adds message to final entity entry on registration form that points to portal dashboard to submit other entity entries manually
…

## Related

- [WP-205](https://jira.tacc.utexas.edu/browse/WP-205)
…

## Changes
- To script that adds new entities to form, checking for the final entry (5th entity) to add message which opens another tab/window to portal dashboard
…

## Testing

1. Go to http://localhost:8000/register/request-to-submit/
2. On the entities portion, click the `+ Add Another Entity` button until you reach the max number of entities
3. On the final entity, confirm there is a message as shown in the image below (link to dashboard will not work unless you have Core-Portal tied to your local repo)

## UI
image

…

### CODE REVIEW

File: apcd-cms/src/apps/registrations/templates/submission_form/registration_form_scripts.html [hunk 1]
```diff
@@ -277,6 +277,12 @@ <h6>
           <div id="help-text-total_claims_value_${entities}" class="help-text">
           </div>
         </div>
+        ${ entities === 5 ?
+            {% if not r %}
+             '<h6>If you need to associate more than 5 entities with your registration, please <a href="/workbench/dashboard" target="_blank">submit a ticket</a> with your additional entries and your registration ID (displayed after submitting this form).</h6>'
```
Comment (wesleyboar) [inline apcd-cms/src/apps/registrations/templates/submission_form/registration_form_scripts.html:282]: This message is not a [heading (`<h1>–<h6>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)). Please use a more appropriate tag.

<details><summary>Styling vs. Semantics</summary>

Choose tag based on semantics, not how it looks when it renders; looks are what CSS is for.

If the most appropriate tag does not have styling suitable to the context, then consider:
- the [message UI component (`c-message`)](https://tacc.utexas.edu/static/ui/components/detail/c-message--type-cms.html)
	<sup>Requires `{% include 'snippets/tup-175-css-alerts-messages-ui-pattern.html' %}` to get `c-message` CSS.</sup>
- a new global class and style for APCD
    <sup>(added to `apcd-cms/src/taccsite_custom/apcd-cms/static/apcd-cms/css/site.cms.css`, perhaps `instructions`)</sup>

</details>

<details><summary>Suggestions</summary>

[**`<p>`**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside)
```suggestion
             '<p class="c-message c-message--type-info c-message--scope-inline">If you need to associate more than 5 entities with your registration, <a href="/workbench/dashboard" target="_blank">submit a ticket</a> with your additional entries and your registration ID (displayed after submitting this form).</p>'
```

[**`<aside>`**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside)
```suggestion
             '<aside class="instructions">If you need to associate more than 5 entities with your registration, <a href="/workbench/dashboard" target="_blank">submit a ticket</a> with your additional entries and your registration ID (displayed after submitting this form).</aside>'
```

[**`<small>`**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small)
```suggestion
             '<small>If you need to associate more than 5 entities with your registration, <a href="/workbench/dashboard" target="_blank">submit a ticket</a> with your additional entries and your registration ID (displayed after submitting this form).</small>'
```

</details>

**Review by wesleyboar — CHANGES_REQUESTED** [review]

**Review by wesleyboar — APPROVED** [review]
Thanks for the update.

<details><summary>Out-of-Scope Notes</summary>

I notice the `c-message--type-info` and `c-message--scope-inline` have no effect, but that's a feature not available in APCD yet. I'll make a PR to main to update message styles.

The`snippets/tup-175-css-alerts-messages-ui-pattern.html` is old placeholder CSS. Core-Styles is mature enough to be able to provide newer styles piecemeal.

</details>

**Review by happycodemonkey — APPROVED** [review]
Looks great

**Review by sophia-massie — APPROVED** [review]
LGTM! Also approved @wesleyboar [#217](https://github.com/TACC/Core-CMS-Custom/pull/217 ) related to this

**Pull request merged**